### DISCUSSION TITLE
I appear to not be running the current version of wix.exe

### DISCUSSION THREAD

**Discussion by cpuwzd** [wixtoolset/issues#7752] [Questions]
When I type "wix --version at a command prompt (actually a PowerShell prompt),
the response I get is:

4.0.0-rc.3+5b1a1547

I've tried updating by using "net tool update wix", but it does not change the output.

Any suggestions?

**Comment by robmen** [comment]
If you installed `wix` globally initially, be sure to add the `-g` switch to the `update` command as well.

**Comment by chrpai** [comment]
I wouldn't feel bad, I had the same issue.  I couldn't figure out the commands either as it would just say installed and I'd get the wrong version still.  So I did an uninstall and an install and problem was solved.
